src/tests/ftest/control/dmg_system_reformat.py

```diff
@@ -51,19 +49,27 @@ def test_dmg_system_reformat(self):
         if self.get_dmg_command().result.exit_status != 0:
             self.fail("Detected issues performing a system stop: {}".format(
                 self.get_dmg_command().result.stderr_text))
+
         # Remove pools
         self.pool = []
 
+        # Perform a dmg system erase to allow the dmg storage format to succeed
+        self.log.info("Perform dmg system erase on all system ranks:")
+        self.get_dmg_command().system_erase()
+        if self.get_dmg_command().result.exit_status != 0:
+            self.fail("Issues performing system erase: {}".format(
+                self.get_dmg_command().result.stderr_text))
+
         # To verify that we are using the membership information instead of the
         # dmg config explicit hostlist
         # Uncomment below after DAOS-5979 is resolved
         # self.assertTrue(
         #     self.server_managers[-1].dmg.set_config_value("hostlist", None))
 
         self.log.info("Perform dmg storage format on all system ranks:")
-        self.get_dmg_command().storage_format(reformat=True)
+        self.get_dmg_command().storage_format(force=True)
         if self.get_dmg_command().result.exit_status != 0:
-            self.fail("Issues performing storage format --reformat: {}".format(
+            self.fail("Issues performing storage format --force: {}".format(
```

Comment:
The test is intended to verify reformat function so any reason to change to --force?

Reply:
When attempting to verify my changes did not adversely affect this test I was attempting to get the test to pass with similar changes to what @mjmac is doing in https://github.com/daos-stack/daos/pull/5203/files.  I was able to verify my pool creation changes but the test still did not pass with the other changes.  Ultimately I restored the `@skipForTicket("DAOS-6004")` and the test is still skipped.